behave/features/1.e2e_journey_no_nhs_login.feature

```diff
@@ -0,0 +1,114 @@
+        Given I am on the "nhs-number" page
+        When I give the "#nhs_number" field the value "1116432455"
+        And I submit the form
+        Then wait "2" seconds
+        Then I am redirected to the "name" page
+
+    Scenario: Should be re-directed to date of birth when first name and last name entered
+        Given I am on the "name" page
+        When I give the "#first_name" field the value "Terry"
+        And I give the "#last_name" field the value "Tester"
+        And I submit the form
+        Then wait "2" seconds
+        Then I am redirected to the "date-of-birth" page
+
+    Scenario: Should be re-directed to address lookup when valid date of birth entered
+        Given I am on the "date-of-birth" page
+        When I give the "#date_of_birth-day" field the value "08"
+        And I give the "#date_of_birth-month" field the value "05"
+        And I give the "#date_of_birth-year" field the value "2006"
+        And I submit the form
+        Then wait "2" seconds
+        Then I am redirected to the "address-lookup" page
+
+    Scenario: Should be redirected to support address when an address is selected
+        Given I am on the "address-lookup" page
+        When I submit the form
+        Then wait "2" seconds
+        Then I am redirected to the "support-address" page
+
+    Scenario: Should be redirected to shopping assistance when a valid address is entered
+        Given I am on the "support-address" page
+        When I submit the form
+        Then wait "2" seconds
+        Then I am redirected to the "do-you-have-someone-to-go-shopping-for-you" page
+
+    Scenario: Should be redirected to priority supermarket deliveries when no answered to shopping help
+        Given I am on the "do-you-have-someone-to-go-shopping-for-you" page
+        When I click the ".govuk-radios__item input[value='0']" element
+        And I submit the form
+        Then wait "2" seconds
+        Then I am redirected to the "priority-supermarket-deliveries" page
+
+    Scenario: Should be redirected to basic care needs when yes answered to priority shopping deliveries
+        Given I am on the "priority-supermarket-deliveries" page
+        When I click the ".govuk-radios__item input[value='1']" element
+        And I submit the form
+        Then wait "2" seconds
+        Then I am redirected to the "basic-care-needs" page
+
+    Scenario: Should be redirected to contact details when no answered to basic care needs help
+        Given I am on the "basic-care-needs" page
+        When I click the ".govuk-radios__item input[value='0']" element
+        And I submit the form
+        Then wait "2" seconds
+        Then I am redirected to the "contact-details" page
+
+    Scenario: Should be redirected to check contact details when email entered
+        Given I am on the "contact-details" page
+        When I give the "#email" field the value "sandeep.nagra@digital.cabinet-office.gov.uk"
```

Comment:
update the value for the email address to be coronavirus-services-smoke-tests@digital.cabinet-office.gov.uk and perhaps put this as a config value